pkg/osquery/tables/execparsers/dpkg/parser_test.go

```diff
@@ -0,0 +1,152 @@
+package dpkg
+
+import (
+	"bytes"
+	_ "embed"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+//go:embed test-data/dpkg_info.txt
+var dpkg_info []byte
+
+func TestParse(t *testing.T) {
+	t.Parallel()
+
+	var tests = []struct {
```

Comment:
same as other parser, would like to see test cases for malformed data, missing data, no data, etc